vispy/visuals/markers.py

```diff
@@ -716,7 +600,8 @@ def set_data(self, pos=None, size=10., edge_width=1., edge_width_rel=None,
                                       ('a_fg_color', np.float32, 4),
                                       ('a_bg_color', np.float32, 4),
                                       ('a_size', np.float32),
-                                      ('a_edgewidth', np.float32)])
+                                      ('a_edgewidth', np.float32),
+                                      ('a_symbol', np.float32)])
```

Comment:
Unfortunately we have to use float and not ints, because shaders don't accept ints as attributes and varyings. I'm not sure why, but using `float16` resulted in weird visual bugs (no markers, just flat squares); I suspect it's because there is no implicit conversion?